src/main/java/de/bonndan/nivio/model/Process.java

```diff
@@ -0,0 +1,50 @@
+package de.bonndan.nivio.model;
+
+import de.bonndan.nivio.assessment.Assessable;
+import de.bonndan.nivio.assessment.StatusValue;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class Process implements Linked, Tagged, Labeled, Assessable {
+
+    private final Map<String, Link> links = new HashMap<>();
+    private final Map<String, String> labels = new HashMap<>();
+
+    @Override
+    public Set<StatusValue> getAdditionalStatusValues() {
+        return null;
+    }
+
+    @Override
+    public String getAssessmentIdentifier() {
+        return null;
+    }
+
+    @Override
+    public List<? extends Assessable> getChildren() {
+        return null;
+    }
+
+    @Override
+    public String getLabel(String key) {
+        return null;
```

Comment:
Returning null is not the best practice.

Reply:
oh sh*t, I accidentally committed this file. 🙈